documents/Libraries/stdlib/stdlib_sx-glsl_impl.mtlx

```diff
@@ -784,10 +784,17 @@
   <!-- ======================================================================== -->
 
   <!-- <blur> -->
+  <implementation name="IM_blur_float_sx_glsl" nodedef="ND_blur_float" function="mx_blur_float" language="sx-glsl"/>
+  <implementation name="IM_blur_color2_sx_glsl" nodedef="ND_blur_color2" function="mx_blur_color2" language="sx-glsl"/>
+  <implementation name="IM_blur_color3_sx_glsl" nodedef="ND_blur_color3" function="mx_blur_color" language="sx-glsl"/>
+  <implementation name="IM_blur_color4_sx_glsl" nodedef="ND_blur_color4" function="mx_blur_color4" language="sx-glsl"/>
+  <implementation name="IM_blur_vector2_sx_glsl" nodedef="ND_blur_vector2" function="mx_blur_vector2" language="sx-glsl"/>
+  <implementation name="IM_blur_vector3_sx_glsl" nodedef="ND_blur_vector3" function="mx_blur_vector" language="sx-glsl"/>
+  <implementation name="IM_blur_vector4_sx_glsl" nodedef="ND_blur_vector4" function="mx_blur_vector4" language="sx-glsl"/>
```

Comment:
For consistency, on the function names we could prefix with the materialx type names "vector3" and "color3", insted of "color" and "vector".

Reply:
Will do.